packages/manager/package.json

```diff
@@ -0,0 +1,127 @@
+{
```

Comment:
We have to bundle some dependencies in the packages due to ESM incompatibilities. Some packages are only published with ESM versions, which are incompatible with CJS applications (at least incompatible without drastically changing the way apps are written and run).

Vite bundles `node_module` dependencies by transpiling them into a `node_modules` directory. We rename `dist/node_modules` to `__node_modules` because npm automatically ignores `node_modules` when publishing.

Reply:
I've seen that issue a few times, how does it know which modules to transpile?